deploy/cdk/lib/static-host/static-host-stack.ts

```diff
@@ -0,0 +1,146 @@
+
+    this.bucket = new s3.Bucket(this, 'SiteBucket', {
+      serverAccessLogsBucket: props.foundationStack.logBucket,
+      serverAccessLogsPrefix: `s3/${this.hostname}`,
+    })
+
+    this.cloudfront = new CloudFrontWebDistribution(this, 'Distribution', {
+      comment: this.hostname,
+      errorConfigurations: [
+        {
+          errorCode: 403,
+          responseCode: 404,
+          responsePagePath: '/404.html',
+        },
+      ],
+      loggingConfig: {
+        bucket: props.foundationStack.logBucket,
+        includeCookies: true,
+        prefix: `web/${this.hostname}`,
+      },
+      originConfigs: [
+        {
+          s3OriginSource: {
+            s3BucketSource: this.bucket,
+            originAccessIdentity: new OriginAccessIdentity(this, 'OriginAccessIdentity', {
+              comment: `Static assets in ${this.stackName}`,
+            }),
+          },
+          behaviors: [
+            {
+              allowedMethods: CloudFrontAllowedMethods.GET_HEAD_OPTIONS,
+              compress: true,
+              defaultTtl: (props.contextEnvName === 'dev') ? cdk.Duration.seconds(0) : cdk.Duration.days(1),
+              isDefaultBehavior: true,
+              lambdaFunctionAssociations: [
+                {
+                  eventType: LambdaEdgeEventType.ORIGIN_REQUEST,
+                  lambdaFunction: this.spaRedirectionLambda.currentVersion,
+                },
+              ],
+            },
+          ],
+        },
+      ],
+      viewerCertificate: ViewerCertificate.fromAcmCertificate(props.foundationStack.certificate, {
+        aliases: [this.hostname],
+        securityPolicy: SecurityPolicyProtocol.TLS_V1_1_2016,
+        sslMethod: SSLMethod.SNI,
+      }),
+      viewerProtocolPolicy: ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
+    })
+
+    // Create DNS record (conditionally)
+    if (props.createDns) {
+      new CnameRecord(this, 'ServiceCNAME', {
+        recordName: this.hostname,
+        comment: this.hostname,
+        domainName: this.cloudfront.distributionDomainName,
+        zone: props.foundationStack.hostedZone,
+        ttl: cdk.Duration.minutes(15),
+      })
+    }
+
+    new ssm.StringParameter(this, 'BucketParameter', {
+      parameterName: `/all/stacks/${this.stackName}/site-bucket-name`,
+      description: 'Bucket where the stack website deploys to.',
+      stringValue: this.bucket.bucketName,
+    })
+
+    new cdk.CfnOutput(this, 'Hostname', {
+      value: props.createDns ? this.hostname : this.cloudfront.distributionDomainName,
+      description: 'The FQDN if one is given, otherwise the cloudfront distribution domain name.',
+      exportName: `${this.stackName}:Hostname`,
+    })
+
+    new cdk.CfnOutput(this, 'BucketName', {
+      value: this.bucket.bucketName,
+      description: 'Name of S3 bucket to hold website content',
+      exportName: `${this.stackName}:BucketName`,
+    })
```

Comment:
Do these need to be exports? I don't find we often need to make exports any longer, unless there's some infra code external to this cdk app that will need it.

Reply:
Maybe not. The old stack did and I'm not privy to whether something is looking for it or not.

Reply:
Gotcha, expected that was the case. Just checked our prod exports, only thing that imports these is the code pipeline for deploying to the bucket and testing the host. Looks like you're passing this info around differently, so should be safe to remove.